mobile-app-automation/readme.md

```diff
@@ -0,0 +1,82 @@
+# Coding task for Mobile QA Automation Engineer @tradeling
+
+## The Goal
+
+Your task is to write automated tests preferably with [Appium](https://appium.io/), or [Detox](https://github.com/wix/Detox/)  to demonstrate your test automation abilities.
+
+
+## The Process
+
+1. Start a new github repository with detailed "README" document on how to execute the tests.
+1. Download Amazon Shopping Android or App Store Package (APK or IPA)
+1. Write end-to-end tests fulfilling the [Acceptance Criteria](#acceptance-criteria) below.
+1. Publish test execution report in html format.
+1. Prepare a test summary report and include the information about platform, app, tools etc
+1. Include all tests related document in same github repository.
+
+- Use below package and activity name (Appium)
+  - appPackage=com.amazon.mShop.android.shopping
+  - appActivity=com.amazon.mShop.home.HomeActivity
+
+## Acceptance Criteria
+
+- Scenario 1: [Check Add a new Payment to your profile]
```

Comment:
1. Open Amazon app with region United State
2. Login with valid credentials
3. Tap on Your account
4. From Payments section, select Your Payments
5. Tap on Wallet and add a credit or debit card as the payment method
6. Assert on added card with correct type [Visa/Mastercard]